examples/tts/radtts.py

```diff
@@ -21,6 +26,12 @@
 from nemo.utils.exp_manager import exp_manager
 
 
+def handle_pdb(sig, frame):
+    import pdb
+
+    pdb.Pdb().set_trace(frame)
```

Comment:
?

Reply:
Debugging hook in case the training gets hung.

Reply:
Removed debugging hooks.